internal/pkg/config/v1/config.go

```diff
@@ -259,14 +267,18 @@ func (c *Config) encryptContextStateTokens(tempAuthToken, tempAuthRefreshToken s
 		c.Context().GetState().Salt = salt
 		c.Context().GetState().Nonce = nonce
 	}
-	if tempAuthToken != "" {
+
+	reg1 := regexp.MustCompile(authTokenRegex)
+	if tempAuthToken != "" && reg1.MatchString(tempAuthToken) {
```

Comment:
If we can't come up with a better name than `reg1`, let's just do this:
```suggestion
	if tempAuthToken != "" && regexp.MustCompile(authTokenRegex).MatchString(tempAuthToken) {
```